### PR TITLE
MNT: Try to import backend_qt rather than backend_qt5

### PR DESCRIPTION
**Pull request by tacaswell**

### CODE REVIEW

File: lib/mplcairo/qt.py [hunk 1]
```diff
@@ -1,7 +1,12 @@
 import ctypes
 
 # This does support QT_API=pyqt6 on Matplotlib versions that can handle it.
-from matplotlib.backends.backend_qt5 import _BackendQT5, FigureCanvasQT
+try:
+    from matplotlib.backends.backend_qt import _BackendQT, FigureCanvasQT
+except ImportError:
+    from matplotlib.backends.backend_qt5 import (
+        _BackendQT5 as _BackendQT, FigureCanvasQT
+    )
```
Comment (anntzer) [inline lib/mplcairo/qt.py:9]: move parenthesis to previous line (if the in-parenthesis contents are not one-item-per-line, the "minimize diff" argument does not apply).

File: lib/mplcairo/qt.py [hunk 2]
```diff
@@ -46,6 +51,9 @@ def blit(self, bbox=None):  # matplotlib#17478 (<3.3).
...
 
 
-@_BackendQT5.export
-class _BackendQT5Cairo(_BackendQT5):
+@_BackendQT.export
+class _BackendQTCairo(_BackendQT):
     FigureCanvas = FigureCanvasQTCairo
+
+# back-compat name
+_BackendQT5Cairo = _BackendQTCairo  # noqa
```
Comment (anntzer) [inline lib/mplcairo/qt.py:59]: not public, no need for backcompat.

**Comment by anntzer** [comment]
Thanks.  Just a few style things :p

**Comment by tacaswell** [comment]
fixed and force-pushed

**Comment by anntzer** [comment]
Thanks.

**Pull request merged**